spec/physical_volume_spec.rb

```diff
@@ -76,17 +76,17 @@
 
   describe "#scan" do
     it "uses pvdisplay" do
-      expect(described_class).to receive(:run!).
-                                 with(LinuxAdmin.cmd(:pvdisplay),
-                                     :params => { '-c' => nil}).
-                                 and_return(double(:output => @physical_volumes))
-      expect(LinuxAdmin::VolumeGroup).to receive(:run!).and_return(double(:output => @groups)) # stub out call to vgdisplay
+      expect(LinuxAdmin::Common).to receive(:run!)
+        .with(LinuxAdmin::Common.cmd(:pvdisplay),
+              :params => {'-c' => nil})
+        .and_return(double(:output => @physical_volumes))
+      expect(LinuxAdmin::Common).to receive(:run!).and_return(double(:output => @groups)) # stub out call to vgdisplay
       described_class.scan
     end
 
     it "returns local physical volumes" do
-      expect(described_class).to receive(:run!).and_return(double(:output => @physical_volumes))
-      expect(LinuxAdmin::VolumeGroup).to receive(:run!).and_return(double(:output => @groups))
+      expect(LinuxAdmin::Common).to receive(:run!).and_return(double(:output => @physical_volumes))
+      expect(LinuxAdmin::Common).to receive(:run!).and_return(double(:output => @groups))
```

Comment:
You should be able to put both doubles in a single `and_return` and the first hit will get the first response and the second will get the second response.